pkg/oc/admin/diagnostics/diagnostics/cluster/network/in_pod/pod.go

```diff
@@ -8,6 +8,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	kapi "k8s.io/kubernetes/pkg/apis/core"
 	kclientset "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
+	kcontainer "k8s.io/kubernetes/pkg/kubelet/container"
```

Comment:
No, you are not allowed to add this package set to `oc`.  Diagnostics may not depend on any apiserver, controller manager, or kubelet code.

/hold